app/views/sessions/new.html.haml

```diff
@@ -10,4 +10,4 @@
           = label_tag :password, "Password"
           = password_field_tag :password
         %fieldset.action
-          %input(type="submit" value="Sign In" class="btn")
\ No newline at end of file
+          %input(type="submit" value="Sign in" class="btn")
```

Comment:
In general, I would use Rails helpers if you are using them before - to remain consistent `submit_tag "Sign in", class: "btn" `.